scripts/fuzz_shell.js

```diff
@@ -417,9 +433,11 @@ if (secondBinary) {
   });
 }
 
-// Compile and instantiate a wasm file.
-function build(binary) {
-  if (secondBinary) {
+// Compile and instantiate a wasm file. Receives the binary to build, and
+// whether it is the second one.
+function build(binary, second) {
```

Comment:
Can we make it more obvious that the second parameter is expected to be a bool? Maybe by calling it something like `isSecond`?

Reply:
Done.